vibe_code_apps_22/ml/eplus_validation_contract.py

```
@@ -0,0 +1,623 @@
+    return df.reset_index(drop=True)
+
+
+def trial_simulated_monthly_kwh(
+    sim_dir: Path,
+    *,
+    heat_cop: float = 3.5,
+    cool_cop: float = 4.5,
+) -> pd.DataFrame:
+    """Aggregate trial proxy kW → monthly kWh (interval-end LST→UTC policy)."""
+    mod = parse_eplus_proxy_to_utc(sim_dir, heat_cop=heat_cop, cool_cop=cool_cop)
+    # 15-min mean kW × 0.25 h = kWh
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)